public/pages/Dashboard/containers/Dashboard.js

```diff
@@ -77,15 +81,17 @@ export default class Dashboard extends Component {
       sortField,
       totalAlerts: 0,
       totalTriggers: 0,
-      chainedAlert: undefined,
+      //chainedAlert: undefined,
```

Comment:
Is this intended to be commented out still? Would this impact composite monitors?